lib/passport/passport.strategy.ts

```diff
@@ -25,12 +25,13 @@ export function PassportStrategy<T extends Type<any> = any>(
         }
       };
 
-      super(...args, callback);
-      if (this.validate) {
+      const validate = new.target.prototype.validate;
```

Comment:
I'm wondering if there's any possibility that this mixin class would be evaluated without `new` keyword leading to `new.target === undefined` case. Maybe we should add ?. just to make sure(?)
```suggestion
      const validate = new.target?.prototype?.validate;
```

Reply:
I can't imagine a situation where someone would need to evaluate mixin class without `new` or where `new.target` does not have a prototype, but I agree that optional chaining operators can be added to make sure this will never break.

Reply:
I'm thinking of a situation in which someone instead of calling `AuthGuard('name)` would accidentally call `AuthGuard('name')()` and that would  lead to an exception `Cannot read property 'prototype' of undefined`

Reply:
Alright, let's add ? just in case!